chat/models.py

```diff
@@ -0,0 +1,27 @@
+from django.db import models
+from django.utils import timezone
+
+
+# class ChatManager(models.Manager):
+#     def get_group_chat(self, group_id):
+#         return self.filter(group_id=group_id).order_by('date_time')
+#
+#     def get_student_course_chat(self, student_course_id):
+#         return self.filter(student_course_id=student_course_id).order_by('date_time')
+
+
+class Message(models.Model):
+    message_id = models.BigAutoField(primary_key=True)
+    # sender_id = models.ForeignKey("User", on_delete=models.CASCADE)
+    # student_course_id = models.ForeignKey("Student_Course", on_delete=models.CASCADE)
+    # group_id = models.ForeignKey("Group", on_delete=models.CASCADE)
+    date_time = models.DateTimeField(default=timezone.now,  blank=False)
```

Comment:
I suggest setting the date_time field to be filled automatically according to the time the message was sent.

Reply:
I changed it :)